ci-operator/config/openshift/verification-tests/openshift-verification-tests-master__installation-nightly-4.16.yaml

```diff
@@ -311,6 +311,15 @@ tests:
     test:
     - chain: cucushift-installer-check-cluster-health
     workflow: cucushift-installer-rehearse-azure-ipi-generation
+- as: azure-ipi-default-mini-perm-f7
```

Comment:
suggest to use `arm` to run testing to save some cost.

BTW, do we need to run it once per 7 days ?  How about f14 ?

Reply:
updated to `arm`. But I still prefer f7 to run once at least one week to catch blocker bugs like https://issues.redhat.com/browse/OCPBUGS-42296

Reply:
okay.